### PR TITLE
added a run once option to launcher

### PR DESCRIPTION
**Pull request by mikeburkat**

### CODE REVIEW

File: launcher/launcher.h [hunk 1]
```diff
@@ -52,8 +52,10 @@ struct Launcher
         }
 
         void restart(std::string const & node) {
```
Comment (wsourdeau) [inline launcher/launcher.h]: Should "restart" be invoked at all for affected nodes?

Comment (mikeburkat) [inline launcher/launcher.h]: well then I would of had to make the var once public and checked in the service, or make an accessor. this seemed simpler, but if it's bad style I can change it... What do you suggest?

Comment (wsourdeau) [inline launcher/launcher.h]: I am not clear. I don't think it matters much actually.

**Comment by EricRobert** [comment]
+1

**Comment by wsourdeau** [comment]
@mikeburkat can this PR be merged?

**Comment by mikeburkat** [comment]
yeah if you want to, but to be honest I haven't needed it since that one time I did this for.

**Pull request merged**